src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -264,13 +270,17 @@ void clearUpstreamDeletionsLoc() {
     }
 
     /**
-     *  Record deletion to keep
-     *  Add deletions to a list.
+     *  Record emitted deletion in order to remove downstream spanning deletion alleles that are not covered by any emitted deletion.
+     *  In addition to recording a new deletion, this method culls previously-recorded deletions that end before the current variant
+     *  context.  This assumes that variants are traversed in order.
      *
      * @param deletionSize  size of deletion in bases
```

Comment:
Can you note that this is the size for the normalized representation?  Otherwise it's not obvious why we need this information in addition to the VC.

Reply:
We really didn't need it.  I rewrote the method to take in the VC and the emitted alleles.